packages/kbn-investigation-shared/src/rest_specs/get_events.ts

```diff
@@ -8,7 +8,7 @@
  */
 
 import { z } from '@kbn/zod';
```

Comment:
What is this none self-explanatory `z` 🤔 😆 ?

Reply:
Did I add that? I don't recall importing from @kbn/zod but that's probably just a wrapper around zod itself

Reply:
https://zod.dev/?id=introduction